qa-tests-backend/src/main/groovy/util/Env.groovy

```diff
@@ -281,14 +281,6 @@ class Env {
         return mustGet("GCP_GCS_BACKUP_TEST_BUCKET_NAME_V2")
     }
 
-    static String mustGetGCPAccessKeyID() {
```

Comment:
Do we have a test for GCS integration?

Reply:
Yes, but it does not seem to require the access key data.